### PR TITLE
Awesome-books-with-modules

### PR DESCRIPTION
**Pull request by toyman640**
- Create new directory
- Create new modules for app functionality
- Install luxon library
- Add luxon date to html page

### CODE REVIEW

File: modules/new-book.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const booksSection = document.querySelector('.display-books');
+const formSection = document.querySelector('.form-section');
+const contactSection = document.querySelector('.contact-section');
+
+function showForm() {
+  booksSection.classList.add('hide');
+  formSection.classList.remove('hide');
+  contactSection.classList.add('hide');
+}
```
Comment (LionRouge1) [inline modules/new-book.js]: - [ ] Nice work there :clap: :sunglasses: But as requested by the project requirement, we should refactor the code using arrow function. Therefore let's change this `function` keyword and use arrow function.

File: modules/show-books.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const booksSection = document.querySelector('.display-books');
+const formSection = document.querySelector('.form-section');
+const contactSection = document.querySelector('.contact-section');
+
+function showBooks() {
+  booksSection.classList.remove('hide');
+  formSection.classList.add('hide');
+  contactSection.classList.add('hide');
+}
```
Comment (LionRouge1) [inline modules/show-books.js]: - [ ] Same here too, as pointed out earlier :point_up_2:

Comment (LionRouge1) [inline modules/show-books.js]: - [ ] Same here too, as pointed out earlier :point_up_2:

File: modules/show-contact.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const booksSection = document.querySelector('.display-books');
+const formSection = document.querySelector('.form-section');
+const contactSection = document.querySelector('.contact-section');
+
+function showContact() {
+  booksSection.classList.add('hide');
+  contactSection.classList.remove('hide');
+  formSection.classList.add('hide');
+}
```
Comment (LionRouge1) [inline modules/show-contact.js]: - [ ] Same here too, as pointed out earlier :point_up_2:

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @toyman640, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Great job dividing your code into reasonable modules :+1: 
- All the functionalities of you app are working well :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Nice work implementing all the functionalities :clap: :sunglasses: However it would be nice to link to the list of books or display a success message when a new book is successfully added for user experience.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**